api/ExportSatellitesRoute.go

```diff
@@ -0,0 +1,70 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/jmoiron/sqlx"
+	"github.com/maritza05/golang-bootcamp-2020/satellites/adapters/repos"
+	"github.com/maritza05/golang-bootcamp-2020/satellites/adapters/rest"
+	"github.com/maritza05/golang-bootcamp-2020/satellites/app"
+)
+
+func RegisterExportSatellitesRoute(router *mux.Router, path string) {
+	dbClient := getDbClient()
+	postgresRepo := repos.NewPostgresRepo(dbClient)
+	csvRepo, err := repos.NewCsvRepo(nil)
+	if err != nil {
+		panic("Error while initializing csv repository")
+	}
+	exportService := app.NewExportSatelliteService(postgresRepo, csvRepo)
+	exportController := rest.NewExportSatelliteController(exportService, new(DefaultFileHandler))
+
+	router.HandleFunc(path, exportController.Export).Methods(http.MethodPost)
+}
+
+type DefaultFileHandler struct {
```

Comment:
This abstraction was created to test the handler without the need to create and handle temporary files.